en/docs/quick-start-guides/configure-front-end-application-qsg.md

```diff
@@ -20,7 +20,10 @@ To configure the front-end application, follow the steps given below:
     | **Field**             | **Description**                               |
     |-----------------------|-----------------------------------------------|
     | **Config Name**       | **Web App Config**                            |
-    | **Mount Path**        | **/usr/share/nginx/html/config.js**. Every config that needs to be exposed through the web server should be placed inside `/usr/share/nginx/html/`                          |
+    | **Mount Path**        | **/app/public/config.js**. Every config that needs to be exposed through the web server should be placed inside `/app/public`  or   `/usr/share/nginx/html/`                      |
+
+!!! note
```

Comment:
Shall we rephrase it as "In static web applications, when a configuration file exists within the public folder of the source and shares the same name as the file mounted to the /app/public mount path, the latter file will take precedence over the one inside the public folder."